source/Calamari/Integration/FileSystem/CalamariPhysicalFileSystem.cs

```diff
@@ -114,7 +117,7 @@ public void DeleteFile(string path, FailureOptions options)
 
         public void DeleteDirectory(string path)
         {
-            Directory.Delete(path, true);
+            Directory.Delete(path);
```

Comment:
I've just hard-coded `true` in the implementation of Delete, since we pass `true` in all our uses. I might change it back though so we keep the same API as the underlying Dir.Delete. I did that with the other functions, but this was one of the first I wrapped.